modules/Components/src/client/LSFileChooserView.html

```diff
@@ -4,8 +4,11 @@
 			<div class="span7 bv_manualFileSelect" style="padding-left:50px;">
 				<span class="btn fileinput-button">
 					<span>Browse Files...</span>
-					<input type="file" name="files[]" multiple>
+					<input type="file" name="files[]">
```

Comment:
Moved the logic to GenericDataParser.coffee so it will remove the multiple attribute when the experiment loader is opened, that way it won't affect other parts interfaces.